sdk/metric/internal/aggregate/sum_test.go

```diff
@@ -37,6 +37,7 @@ func testSum[N int64 | float64](t *testing.T) {
 		MeasurementN: defaultMeasurements,
 		CycleN:       defaultCycles,
 	}
+	totalMeasurements := defaultGoroutines * defaultMeasurements
```

Comment:
This test produces a _ton_ of duplicate measurements within a single Aggregation.  Since these now sum (instead of keeping the last value), I scaled the expected value of precomputed tests by the number of measurements that go into them.